tests_new/integration_tests/modules/share_base/conftest.py

```diff
@@ -253,15 +253,15 @@ def persistent_consumption_role_1(client5, group5, persistent_cross_acc_env_1, p
 def persistent_group_share_1(
     client5,
     client1,
-    updated_persistent_env1,
-    updated_persistent_cross_acc_env_1,
+    persistent_env1,
```

Comment:
why don't we use updated here? We want to check our persistent shares after envs are updated. Don't we?

Reply:
I dropped the notion of "updated_environnment" and I now force-update an environment the first time is being requested in a test session.